### DISCUSSION TITLE
Allow the use of 🙏 emoji as issue (comment) reaction to express gratitude

### DISCUSSION THREAD

**Discussion by claudio-salvio** [community/community#53370] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

The action that the 🙏 emoji represents is understood in many cultures as an expression of gratitude and I consider that it has become -by far- the most used emoji to express that feeling.
It seems to me that it would be very convenient to add this reaction, given the importance of the expression of gratitude in human interactions, both for the one who gives it and for the one who receives it.

Kind regards,
Claudio Salvio

**Comment by cohhi** [comment]
I agree with your point of view 😄

**Comment by stainlessdang** [comment]
🤝

**Comment by fauzirfann** [comment]
Absolutely, I completely agree with you
